transport_nantes/dashboard/templates/dashboard/email_campaign_details.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'dashboard/email_campaigns.html' %}
```

Comment:
The dashboard app may not have been a good idea -- perhaps its very tiny content should have gone in `asso_tn`.  But things that are specific to other apps surely should stay with those other apps.

So this PR probably shouldn't touch dashboard except to provide a link from the dashboard page to the TB dashboard(s).  The point of apps is to provide abstraction and containment.

Think what you would do if you decided an app was really useful and should be shared with others.  Or think of other apps you've used.  You'd include directions to add a couple hooks into it, mostly via a `urls.py` inclusion and a clearly documented API for calling it from outside the app.  But, otherwise, everything needed would be in one place.